rootcerts.go

```diff
@@ -15,9 +16,12 @@ import (
 // Go's standard library to load system certs.
 type Config struct {
 	// CAFile is a path to a PEM-encoded certificate file or bundle. Takes
-	// precedence over CAPath.
+	// precedence over Certificate and CAPath.
 	CAFile string
 
+	// Certificate is a PEM-encoded certificate. Takes precedence over CAPath.
+	Certificate string
```

Comment:
I have few personal nit-picks here, but not sure if they are wildhead beliefs:
* for consistency with other fields, I'd prefix it with `CA`
* crypto libraries typically pass certs/private keys as []bytes rather than string.  Given that we convert to bytes in `AppendCertificate`, I'd probably mark this `[]byte` and let clients have more flexibility in how to store cert values
* I think this supports specifying a bundle multiple root certificates in pem format.  If so, I'd note it in comments.